api/nameguard/web_api.py

```diff
@@ -273,6 +273,38 @@ async def fake_eth_name_check_get(
     token_id = validate_token_id(token_id)
     return await ng.fake_eth_name_check(network_name=network_name, contract_address=contract_address, token_id=token_id)
 
+class FakeETHNameCheckFieldsRequest(BaseModel):
+    network_name: NetworkName
+    contract_address: str = Path(examples=['0x495f947276749ce646f68ac8c248420045cb7b5e'],
+                                 description='Contract address for the NFT contract (ERC721 and ERC1155 supported).')
+    token_id: str = Path(examples=['61995921128521442959106650131462633744885269624153038309795231243542768648193'],
```

Comment:
We are not using token_id in the new endpoint. We can use it to check if it matches ENS name.